databricks_cli/dbfs/api.py

```diff
@@ -115,13 +116,24 @@ def get_file(self, dbfs_path, dst_path, overwrite, headers=None):
         length = file_info.file_size
         offset = 0
         with open(dst_path, 'wb') as local_file:
+            retries_left = 20
             while offset < length:
-                response = self.client.read(dbfs_path.absolute_path, offset, BUFFER_SIZE_BYTES,
-                                            headers=headers)
-                bytes_read = response['bytes_read']
-                data = response['data']
-                offset += bytes_read
-                local_file.write(b64decode(data))
+                try:
```

Comment:
I think this logic should be moved to `DbfsClient`, or even to `ApiClient`. `GET` are supposed to be idempotent and the client can configure the retries.